incuna_auth/middleware/permission_feincms.py

```diff
@@ -35,7 +35,10 @@ def _get_page_from_path(self, path):
         Override this to deal with different types of object from Page.
         """
         from feincms.module.page.models import Page
-        return Page.objects.best_match_for_path(path)
+        try:
+            return Page.objects.best_match_for_path(path)
+        except Page.DoesNotExist:
+            return None
```

Comment:
Would that be better to add the `try...except` in [`_get_resource_access_state`](https://github.com/incuna/incuna-auth/pull/42/files#diff-db9cf02d3b0c7e217bc83bc831741fcfR53)

```python
     def _get_resource_access_state(self, request):
         try:
             feincms_page = self._get_page_from_path(request.path_info.lstrip('/'))
         except Page.DoesNotExist:
             return None
```